recipes/lrez/build.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
-export HTSLIB_ROOT=${PREFIX}/lib
-rm -rf build
-mkdir build
-cd build
-cmake ..
-make -j2
-cd ..
 
-mkdir -p $PREFIX/bin/
 
+export C_INCLUDE_PATH=${PREFIX}/include
+export CPLUS_INCLUDE_PATH=${PREFIX}/include
+export LD_LIBRARY_PATH=${PREFIX}/lib
+export LIBRARY_PATH=${PREFIX}/lib
+
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="-lz -lm -lc -lboost_iostreams -L$PREFIX/lib"
+export CPATH=${PREFIX}/include
 
-cp build/tools/Compare ${PREFIX}/bin/
-chmod +x $PREFIX/bin/Compare
+mkdir -p ${PREFIX}/bin
 
-cp build/tools/BamExtractor ${PREFIX}/bin/
-chmod +x $PREFIX/bin/BamExtractor
+# Install bamtools
+cd bamtools
+mkdir -p build
+cd build
+cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$PWD/../ ..
+make
+make install
+cd ../
+if [ -d "lib64" ]
+then
+	mv lib64 lib
+fi
+cp ${SRC_DIR}/bamtools/lib/*.so* ${PREFIX}/lib
 
-cp scripts/reads_bx_sqlite3.py ${PREFIX}/bin/
-chmod +x $PREFIX/bin/reads_bx_sqlite3.py
+# Install LRez
+cd ../
+make CC="$CXX -L${PREFIX}/lib"
```

Comment:
Ideally `-L` should be used via `LDFLAGS`.
If you really need to inject them this way, rather use `"${CXX} ${LDFLAGS}"` instead.